lib/InstallerUtils.py

```diff
@@ -32,10 +32,23 @@
 import prettytable
 
 from lib.InstallLogger import get_install_logger
+from cray_product_catalog.query import ProductCatalog
 
 install_logger = get_install_logger(__name__)
 # pylint: disable=consider-using-f-string
 
+def highestVersion(versions_list):
+    parsed_versions = []
+    for version in versions_list:
+        try:
+            parsed_versions.append(Version.parse(version))
+        except ValueError:
+            install_logger.debug("Found invalid version: %s", version)
```

Comment:
I assume in this context the invalid version will simply not be considered.  It would be good if the logger message could say that, like `"Ignoring invalid version: %s", version`